test/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDaoTest.java

```diff
@@ -475,6 +477,36 @@ public void getAllSurveysMostRecentVersion() throws Exception {
     }
 
     @Test
+    public void canGetSummarySurvey() throws Exception {
+        // First, it returns no responses without error
+        List<Survey> surveys = surveyDao.getSurveysSummary(studyIdentifier);
+        assertEquals(0, surveys.size());
```

Comment:
This relies on the survey table being empty. This will break if tests are run in parallel on the same stack. (Which is true of all Travis builds and integration tests.) Instead of testing that the survey table is empty, you should simply test that the call doesn't fail (or don't include this part of the test at all).